.yarn/versions/cbf96855.yml

```diff
@@ -0,0 +1,2 @@
+releases:
+  "@yarnpkg/plugin-essentials": patch
```

Comment:
You also need to bump the CLI (`prerelease`) because `@yarnpkg/plugin-essentials` is included inside the bundle that lives in `@yarnpkg/cli`. Other than that, you should decline everything else (including dependencies of the CLI.)